modules/ROOT/pages/branded_ios_app/publishing_ios_app_10.adoc

```diff
@@ -0,0 +1,52 @@
+= Additional Server Configuration 
+:apple-universal-link-support-url: https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html
+:admin_manual: https://doc.owncloud.com/server/administration_manual/index.html
+:apple-app-site-association-file-url: https://developer.apple.com/documentation/security/password_autofill/setting_up_an_app_s_associated_domains
+
+== Add Support for Apple Universal Links
+
+There's some special changes that need to be made. 
+Quoting from Apple's official documentation on {apple-universal-link-support-url}[Universal Link Support]:
+
+[quote]
+____
+Adding support for universal links is easy. 
+There are three steps you need to take:
+
+. Create {apple-app-site-association-file-url}[an apple-app-site-association file] that contains JSON data about the URLs that your app can handle.
+. Upload the apple-app-site-association file to your HTTPS web server. You can place the file at the root of your server or in the `.well-known` subdirectory.
+. Prepare your app to handle universal links. 
+____
+
+The `apple-app-site-association` will be generated by ownbrander, and needs to be served statically on your site through HTTPS. 
```

Comment:
The `apple-app-site-association`...
Not clear what this is. Is this a link, a file, some content?
When generated by ownbrander, where does it get written to? configurable? needs to be copied ?...

Reply:
Fair call. I'll find out more.

Reply:
I linked to the ownBrander section of the documentation. That section I'm not too familiar with, but I would expect regular users to be.